storage/store/merge.go

```diff
@@ -12,6 +12,21 @@ import (
 	pbsubstreams "github.com/streamingfast/substreams/pb/sf/substreams/v1"
 )
 
+func (b *baseStore) setKV(k string, v []byte) {
+	if prev, ok := b.kv[k]; ok {
```

Comment:
This does add a _second_ lookup though; 
The caller of `setKV` often did it itself.

We can live with it for now I think.